netutils/rexec/rexec.c

```diff
@@ -45,8 +46,8 @@ struct rexec_arg_s
   FAR const char *command;
   FAR const char *user;
   FAR char *host;
+  sa_family_t af;
```

Comment:
why the position in the structure is changed?

Reply:
Make it sort by length?